### PR TITLE
Replace accordions with figma design

### PR DESCRIPTION
**Pull request by YazidZidane**

### CODE REVIEW

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://74.179.200.84:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://74.179.200.84:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

File: package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
     "redux": "^4.2.1",
     "redux-devtools-extension": "^2.13.9",
     "sass-mediaqueries": "^1.6.1",
-    "shared-react-components": "github:precision-sustainable-ag/shared-react-components#feature/figmaTextfield",
+    "shared-react-components": "github:precision-sustainable-ag/shared-react-components#feature/figmaAccordion",
```
Comment (Adamws33) [inline package.json:29]: this line is incorrect it should always be     
`"shared-react-components": "github:precision-sustainable-ag/shared-react-components",`

Comment (YazidZidane) [reply package.json:29]: This is just for reviewing, after the code is reviewed and the changes are merged in the shared repo it should be updated to the main branch

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://74.179.200.84:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://74.179.200.84:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Review by Adamws33 — APPROVED** [review]

**Pull request merged**